### PR TITLE
Feature products

### PR DESCRIPTION
**Pull request by RachelAnnB**
-added content to products section
-made products section responsive
-made other small changes

### CODE REVIEW

**Comment by dflynn15** [comment]
This looks really good!

My only suggestion would be to pull out the commented breakpoints in your SASS and assign them as common variables like `$bp-nameorthing` and share those across files. That way if anything changes you wouldn't have to look through multiple files trying to make sure you found all instances of the value you are changing.

**Comment by RachelAnnB** [comment]
Originally I had the break points as variables, but they were named small, medium, large, etc., so Zastrow pointed out that that naming convention can break down pretty quickly, so he suggested just removing the variables.

File: src/scss/_products.scss [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+}
+
+.item__heading__box {
+	width: 100%;
+	background-color: $color-orange;
+	padding: 0.5em 0; //8px 0
+	position: relative;
+	margin: auto;
+	z-index: 1;
+	top: 240px;
```
Comment (aspencer116) [inline src/scss/_products.scss:84]: I believe this could be positioned using em's

Comment (RachelAnnB) [inline src/scss/_products.scss:84]: Do you mean the top: 240px? Or something else?

**Comment by aspencer116** [comment]
It looks good to me! I left one comment on the code.

File: src/scss/_products.scss (hunk 1 above)
Comment (aspencer116) [inline src/scss/_products.scss:84]: Yes

**Pull request merged**